projects/addon-commerce/components/input-card-group/input-card-group.component.ts

```diff
@@ -55,7 +53,8 @@ import {
     tuiDropdownOptionsProvider,
     TuiWithDropdownOpen,
 } from '@taiga-ui/core/directives/dropdown';
-import {TUI_COMMON_ICONS} from '@taiga-ui/core/tokens';
+import type {TuiDataListHost} from '@taiga-ui/core/tokens';
```

Comment:
We need to provide `tuiAsAuxiliary(...)` (from `@taiga-ui/core/components/textfield`) inside `TuiDataList`.
Also, we need to provide `tuiAsDataListHost` (from `@taiga-ui/core/components/data-list`) inside `TuiTextfield`.

`@taiga-ui/core/components/textfield` <=> `@taiga-ui/core/components/data-list` – circular dependency